spec/support/helpers/services_request_helpers.rb

```diff
@@ -10,14 +10,28 @@ def stub_rummager(content_id)
     allow(Services.rummager).to receive(:search).with(search_params).and_return(rummager_results)
   end
 
+  def stub_rummage_with_doc_type(content_id, document_type)
```

Comment:
Typo! I think you meant `stub_rummager_with_doc_type`.

Reply:
Yeah!! Good spot.